src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/resource/openshift/OpenShiftReplicas.kt

```diff
@@ -0,0 +1,98 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.intellij.kubernetes.model.resource.openshift
+
+import com.redhat.devtools.intellij.kubernetes.model.context.IActiveContext
+import com.redhat.devtools.intellij.kubernetes.model.resource.NonCachingSingleResourceOperator
+import com.redhat.devtools.intellij.kubernetes.model.resource.kubernetes.KubernetesReplicas
+import io.fabric8.kubernetes.api.model.HasMetadata
+import io.fabric8.kubernetes.api.model.Pod
+import io.fabric8.kubernetes.api.model.ReplicationController
+import io.fabric8.openshift.api.model.DeploymentConfig
+
+class OpenShiftReplicas(operator: NonCachingSingleResourceOperator, getAllResources: ResourcesRetrieval)
```

Comment:
Another instance of `operator` as a variable name

Reply:
corrected